### PR TITLE
TW-700 Deposit part

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

**Review by svjatoslavtt — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by svjatoslavtt — APPROVED** [review]

File: ios/TempleWallet.xcodeproj/project.xcworkspace/contents.xcworkspacedata
Comment (lendihop) [inline ios/TempleWallet.xcodeproj/project.xcworkspace/contents.xcworkspacedata:1]: do you need this file?

File: ios/TempleWallet.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
Comment (lendihop) [inline ios/TempleWallet.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: and this?

File: ios/TempleWallet.xcodeproj/project.xcworkspace/contents.xcworkspacedata
Comment (keshan3262) [reply ios/TempleWallet.xcodeproj/project.xcworkspace/contents.xcworkspacedata:1]: Sorry, I did it by accident

**Review by svjatoslavtt — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
ios/android build: 1.15_808

**Pull request merged**